### PR TITLE
Update project instructions

### PR DESCRIPTION
**Pull request by jelliotartz**
## PR Overview
- Instructions/explanatory text for an org's projects (items relating to projects on issue #109) are now displayed beneath the 2 section headers (AFFILIATED PROJECTS and ADD PROJECTS TO THIS ORGANIZATION)
  - I think having each of the sentences as separate lines (basically more like bullet points) makes them easier to read, as opposed to a series of sentences in one solid block of text. Would love design input from others about this, but I worked according to this principle.
  - Created a class, `organization-section-instructions`, to add some spacing between each of the lines of text.
  - Created `formatInstructions()` to add HTML elements with the `organization-section-instructions` class, rather than having a bunch of `<div className="organization-section-instructions">` elements in the jsx.
    - Currently each set of instructions is created as an array of strings, which are then formatted via `formatInstructions()`.
    - Potential issues: 
      - Is there a better strategy to accomplish this? I like the simplicity of `formatInstructions()`, and the fact that it keeps the instruction text outside of `render`, but I'm not sure if there's a more efficient or more logical way to get formatted instruction text displayed line-by-line in `ProjectsContainer`
- Created a CSS class `resources-list__status` to isolate and move statusMessage a little to the right, (it seemed like it was hugging the `display_name` field a little closely), and to increase the font size a little.
  - I realize this is somewhat outside of the scope of a PR that is focused on instruction details, hope that's not blocker.
- Towards #109.
- Staged [here](https://update-project-instructions.lab-preview.zooniverse.org/organizations/5/projects).

## PR Status
- Ready for review. 
- I realize there are a few design decisions implemented here (eg, instructions separated by lines, adjusting the statusMessage styling) that might require input from either researchers or our design team. Happy to have input from others and make changes accordingly!

### CODE REVIEW

**Comment by jelliotartz** [comment]
Thanks, I did use `<ul>` and `<li>`s first, but thought it was cleaner to put it all into text strings in arrays and use Javascript for formatting. But I see your point about it being unnecessarily complex to use Javascript for an HTML/CSS task, and will refactor it to just HTML and CSS :)

**Comment by eatyourgreens** [comment]
One note in favour of using better HTML here: it does help VoiceOver decide how to pause properly, and announce elements, when reading through the block of text.

**Comment by srallen** [comment]
Yes, divs and spans should only be used when no other semantic html fits. This is an older article, but a good explanation when HTML 5 was rolling out: http://html5doctor.com/you-can-still-use-div/

With React, we have a lot of filler divs and spans wrapping content because jsx is technically just javascript and you have to correctly return with the function calls to create the elements. React 16 aka React fiber is going to fix this.

File: src/styles/components/organization-layout.styl [hunk 1]
```diff
@@ -12,10 +12,17 @@
...
   color: GREY_MID
   text-transform: uppercase
   margin: 0.2em 0
   text-decoration: none
+
+.organization-section-instructions
+  list-style-type: none
+  margin-left: 0
+
+  li
```
Comment (srallen) [inline src/styles/components/organization-layout.styl]: And here.

File: src/modules/common/components/resources-list.jsx [hunk 2]
```diff
@@ -52,7 +51,7 @@ const ResourcesList = ({
                   <small>{`by ${resource.links.owner.display_name}`}</small>}
                 {showStatus &&
                   <span
-                    className={`color-label ${resource.launch_approved ? 'green' : 'red'}`}
+                    className={`resources-list__status color-label ${resource.launch_approved ? 'green' : 'red'}`}
```
Comment (srallen) [inline src/modules/common/components/resources-list.jsx]: FYI, `classnames` is an installed dependency of the project and is really handy for setting conditional classes. 

Also, using BEM convention, it should be `color-label--red` or `color-label--green`

Out of scope of this PR though.

**Comment by jelliotartz** [comment]
Thanks @srallen, happy to fix the `getMeta()` and `classnames` issues in a subsequent PR :)

**Comment by jelliotartz** [comment]
re: caps statuses, they actually were already styled as all-caps [via CSS](https://github.com/zooniverse/pfe-lab/blob/master/src/styles/global/common.styl#L113), so I just adjusted the text to lowercase to let the CSS do the work.

File: src/modules/organizations/components/organization-projects-list.jsx [hunk 3]
```diff
@@ -9,15 +9,28 @@ const OrganizationProjectsList = ({ projects, onRemove }) => {
...
-    />
+    <div>
+      <h3 className="organization-layout__section-header">Affiliated Projects</h3>
+      <ul className="organization-layout__section-instructions">
+        <li className="organization-layout__section-instructions--list-items">
+          If you see CONTACT ORGANIZATION OWNER as the owner, contact the organization owner,
+          or collaborators, to get access to this project.
+        </li>
+        <li className="organization-layout__section-instructions--list-items">The visibility of each project is displayed to the right of the project&apos;s owner.</li>
+        <li className="organization-layout__section-instructions--list-items">Only the owner and collaborators can view a NOT PUBLICLY VISIBILE project.</li>
```
Comment (mcbouslog) [inline src/modules/organizations/components/organization-projects-list.jsx]: Only the project collaborators can view a NOT PUBLICLY VISIBILE project.

File: src/modules/organizations/components/organization-projects-list.jsx [hunk 4]
```diff
@@ -9,15 +9,28 @@ const OrganizationProjectsList = ({ projects, onRemove }) => {
...
+        <li className="organization-layout__section-instructions--list-items">Anyone with the URL can access a PUBLICLY VISIBILE project.</li>
```
Comment (mcbouslog) [inline src/modules/organizations/components/organization-projects-list.jsx]: This is misleading, see related comment in `resources-list`, I think confusing project attributes of `private` vs `launched_approved`?

File: src/modules/organizations/containers/projects-container.jsx [hunk 5]
```diff
@@ -51,7 +51,7 @@ class ProjectsContainer extends React.Component {
 
       organization.get('projects', query)
         .then((projects) => {
-          this.setState({ meta: projects[0]._meta });
+          this.setState({ meta: projects[0]._meta }); // eslint-disable-line no-underscore-dangle
```
[the remark this answers is not in the document]
Comment (jelliotartz) [reply src/modules/organizations/containers/projects-container.jsx]: Will be addressed in issue #120.

**Pull request merged**